src/Microsoft.Tye.Extensions/Dapr/DaprExtension.cs

```diff
@@ -20,12 +20,67 @@ public override Task ProcessAsync(ExtensionContext context, ExtensionConfigurati
 
             if (context.Operation == ExtensionContext.OperationKind.LocalRun)
             {
+                // default placement port number
+                var placementPort = 50005;
```

Comment:
Can this be a random port instead of a specific port?

Reply:
It will now use the next available port, same as the other parts of the system.